test/type_prop/pad.cpp

```diff
@@ -602,58 +602,6 @@ TEST(type_prop, pad_v1_arg_pads_end_incompatible_type)
     }
 }
 
-TEST(type_prop, pad_v1_pads_begin_elem_negative_value)
```

Comment:
Why are these tests removed?

Reply:
`ONNX` standard requires support for negative indexes (it means removing elements at the beginning and end of each axis).